### PR TITLE
Update Unit of Measurements with new Units from openhab-core

### PR DESCRIPTION
**Pull request by Cossey**
@cweitkamp requested that I update the documentation to reflect the new Unit of Measurements currently available in openHAB core.

_Someone may want to check the stylizing of some of the unit names._

**Note: I have a Pull Request open for openhab/openhab-core#990 which corrects two unit symbols I found when documenting the new Units.**

Signed-off-by: Stewart Cossey <stewart.cossey@gmail.com>

### CODE REVIEW

File: concepts/units-of-measurement.md [hunk 1]
```diff
@@ -97,63 +97,93 @@ SI:
...
+| ElectricCapacitance    | Farad                            | F      |
+| ElectricCharge         | Coulomb                          | C      |
+| ElectricConductance    | Siemens                          | S      |
+| ElectricCurrent        | Ampere                           | A      |
+| ElectricInductance     | Henry                            | H      |
+| ElectricResistance     | Ohm                              | Ω      |
+| Energy                 | Joule                            | J      |
+| Energy                 | Watt Second                      | Ws     |
+| Energy                 | Watt Hour                        | Wh     |
+| Energy                 | KiloWatt Hour                    | kWh    |
```
Comment (cweitkamp) [inline concepts/units-of-measurement.md]:
```suggestion
| Energy                 | Kilowatt Hour                    | kWh    |
```

File: concepts/units-of-measurement.md [hunk 2]
```diff
@@ -97,63 +97,93 @@ SI:
...
+| Energy                 | Megawatt Hour                    | MWh    |
+| Energy                 | Kilovar Hour                     | kvarh  |
+| Force                  | Newton                           | N      |
+| Frequency              | Hertz                            | Hz     |
+| Illuminance            | Lux                              | lx     |
+| Intensity              | Irradiance                       | W/m²   |
+| Intensity              | microWatt per square centimeter  | µW/cm² |
```
Comment (cweitkamp) [inline concepts/units-of-measurement.md]:
```suggestion
| Intensity              | Microwatt per square Centimeter  | µW/cm² |
```

File: concepts/units-of-measurement.md [hunk 3]
```diff
@@ -97,63 +97,93 @@ SI:
...
+| Length                 | Kilometre                        | km     |
+| LuminousFlux           | Lumen                            | lm     |
+| LuminousIntensity      | Candela                          | cd     |
+| MagneticFlux           | Weber                            | Wb     |
+| MagneticFluxDensity    | Tesla                            | T      |
+| Mass                   | Kilogram                         | kg     |
+| Mass                   | Gram                             | g      |
+| Power                  | Watt                             | W      |
+| Power                  | Kilovar                          | kvar   |
+| Power                  | Decibel-milliwatts               | dBm    |
```
Comment (cweitkamp) [inline concepts/units-of-measurement.md]:
```suggestion
| Power                  | Decibel-Milliwatts               | dBm    |
```

File: concepts/units-of-measurement.md [hunk 4]
```diff
@@ -97,63 +97,93 @@ SI:
...
+| Pressure               | Pascal                           | Pa     |
+| Pressure               | hectoPascal                      | hPa    |
```
Comment (cweitkamp) [inline concepts/units-of-measurement.md]:
```suggestion
| Pressure               | Hectopascal                      | hPa    |
```

File: concepts/units-of-measurement.md [hunk 5]
```diff
@@ -97,63 +97,93 @@ SI:
...
+| Pressure               | Millimetre of Mercury            | mmHg   |
+| Pressure               | Bar                              | bar    |
+| Pressure               | milliBar                         | mbar   |
```
Comment (cweitkamp) [inline concepts/units-of-measurement.md]:
```suggestion
| Pressure               | Millibar                         | mbar   |
```

**Comment by Cossey** [comment]
Will correct the styling of the items you’ve pointed out. Regarding Binary prefixes, I’ll add this to the prefixes table. 

_The standard gravity unit symbol (the g with the subscript style “n”) doesn’t appear to render correctly on Safari iOS when looking at the preview URL on my phone. This was copied straight from the source code. I wonder if there might be issues with this particular character on other platforms._

**Comment by Cossey** [comment]
Unit styling all fixed up now. I ended up creating another table specifically for the Binary Prefixes since they reside in a different class. Also changed the Column Value to Factor for the Prefix tables and also mentioned the BinaryPrefixes classes. Also some slight formatting changes and another example showing a Binary Prefix.

*Used this to double-check the changes: https://physics.nist.gov/cuu/Units/binary.html*

**Comment by ThomDietrich** [comment]
Can confirm the subscript n issue. It is neither rendered on GitHub nor in the netlify preview on macos and Firefox.

image

I'm a big fan of unicode symbols but using this one doesn't seem to be a good idea :-/ I propose to go with `g<sub>n</sub>` g<sub>n</sub>

**Comment by Cossey** [comment]
Does anyone else have any other proposals regarding this character? Should this character also be changed in `openHAB-core` as one would assume rendering issues in PaperUI due to the particular Unicode character used?

**Comment by Cossey** [comment]
@cweitkamp This seems stuck until I can get some feedback from others; may also need to open a PR for changing this UoM symbol in openhab-core depending on what we go with.

Happy to wait for further feedback from other relevant parties.

**Comment by cweitkamp** [comment]
Maybe we can ask @Hilbrand for his opinion as he introduced standard gravity in https://github.com/openhab/openhab-core/pull/617. It looks like we are using a different (Unicode) character in OHC. I would suggest to add a unit test over there to verify if it is really working before we change it.

**Comment by Hilbrand** [comment]
This is what the page shows when I open it in Firefox:

image

However. I noticed the `g` seems to be different from what is in openHAB core:

where  | unicode | visual
-- | -- | --
core | g\&#x2099;  | g&#x2099;
this pr |  \&#x0261;\&#x2099; | &#x0261;&#x2099;

Using `<sub>` assumes the context is always HTML. I don't know if that is true.

**Comment by Cossey** [comment]
Hmm okay let me look into this in the next few days so we can get this sorted and issue closed.

**Comment by Cossey** [comment]
> This is what the page shows when I open it in Firefox:
> 
> image
> 
> However. I noticed the `g` seems to be different from what is in openHAB core:
> 
> where	unicode	visual
> core	g&#x2099;	gₙ
> this pr	&#x0261;&#x2099;	ɡₙ
> Using `<sub>` assumes the context is always HTML. I don't know if that is true.

Looking at this Unicode 0261 is the lowercase g so the subscript `n` in question is still the same. Still unsure how to proceed since it appears that many fonts don't display this character.

Maybe we have to settle to using a standard `n` character?

**Comment by cweitkamp** [comment]
Just a stupid guess: What about using an image?

**Comment by Cossey** [comment]
@cweitkamp I think until more system fonts support this unicode character your suggestion is the best way forward and I will do this. 
However how would core handle this as this would be a problem in Paper UI and classic UI too? Should I open an issue there for discussion or just leave it until someone raises a problem with it? AFAIK no one uses this UoM yet.

**Comment by cweitkamp** [comment]
@Cossey I will test it.

**Comment by cweitkamp** [comment]
I am late ...

channel type:
```xml
       <channel-type id="gravity">
               <item-type>Number:Acceleration</item-type>
               <label>Gravity</label>
               <description>Gravity.</description>
               <state readOnly="true" pattern="%.1f %unit%" />
       </channel-type>
```

Java code:
```java
        updateState(channelUID, new QuantityType<>(1, SmartHomeUnits.STANDARD_GRAVITY));
```

Paper UI:

grafik

Basic UI:

grafik

File: concepts/units-of-measurement.md [hunk 6]
```diff
@@ -95,95 +97,135 @@ Imperial:
...
+| Pressure               | Hectopascal                      | hPa    |
+| Pressure               | Millimetre of Mercury            | mmHg   |
+| Pressure               | Bar                              | bar    |
+| Pressure               | Millibar                         | mbar   |
+| Radioactivity          | Becquerel                        | Bq     |
+| RadiationDoseAbsorbed  | Gray                             | Gy     |
+| RadiationDoseEffective | Sievert                          | Sv     |
+| SolidAngle             | Steradian                        | sr     |
+| Speed                  | Metre per Second                 | m/s    |
+| Speed                  | Kilometre per Hour               | km/h   |
```
Comment (cweitkamp) [inline concepts/units-of-measurement.md]: Can we add `mm/h` (see https://github.com/openhab/openhab-core/issues/1297)?

**Comment by Confectrician** [comment]
I am just waiting for a "go" or "pr is ready now" :)

**Comment by Cossey** [comment]
Okay let me deal with #1297 and then it should be good to go.

File: concepts/units-of-measurement.md (hunk 6 above)
Comment (Cossey) [reply concepts/units-of-measurement.md]: Done.

**Comment by Cossey** [comment]
Sorry for the outstanding PR - if it looks good to yous then LGTM?

**Pull request merged**